src/mgmt/tests/srv_drpc_tests.c

```diff
@@ -786,13 +786,154 @@ test_drpc_list_pools_success_with_pools(void **state)
 	D_FREE(resp.body.data);
 }
 
+/*
+ * dRPC Pool SetProp setup/teardown
+ */
+
+static int
+drpc_pool_set_prop_setup(void **state)
+{
+	mock_ds_mgmt_pool_set_prop_setup();
+
+	return 0;
+}
+
+static int
+drpc_pool_set_prop_teardown(void **state)
+{
+	mock_ds_mgmt_pool_set_prop_teardown();
+
+	return 0;
+}
+
+/*
+ * dRPC Pool SetProp tests
+ */
+static void
+test_drpc_pool_set_prop_bad_request(void **state)
+{
+	Drpc__Call	call = DRPC__CALL__INIT;
+	Drpc__Response	resp = DRPC__RESPONSE__INIT;
+	uint8_t		bad_bytes[16];
+	size_t		i;
+
+	/* Fill out with junk that won't translate to a PoolSetPropReq */
+	for (i = 0; i < sizeof(bad_bytes); i++)
+		bad_bytes[i] = i;
+
+	call.body.data = bad_bytes;
+	call.body.len = sizeof(bad_bytes);
+
+	ds_mgmt_drpc_pool_set_prop(&call, &resp);
+
+	assert_int_equal(resp.status, DRPC__STATUS__FAILURE);
+	assert_null(resp.body.data);
+	assert_int_equal(resp.body.len, 0);
+}
+
+static void
+setup_pool_set_prop_drpc_call(Drpc__Call *call, Mgmt__PoolSetPropReq *req)
+{
+	size_t			len;
+	uint8_t			*body;
+
+	len = mgmt__pool_set_prop_req__get_packed_size(req);
+	D_ALLOC(body, len);
+	assert_non_null(body);
+
+	mgmt__pool_set_prop_req__pack(req, body);
+
+	call->body.data = body;
+	call->body.len = len;
+}
+
+static void
+expect_drpc_pool_set_prop_resp_with_error(Drpc__Response *resp,
+                                          int expected_err)
+{
+	Mgmt__PoolSetPropResp *set_prop_resp = NULL;
+
+	assert_int_equal(resp->status, DRPC__STATUS__SUCCESS);
+	assert_non_null(resp->body.data);
+
+	set_prop_resp = mgmt__pool_set_prop_resp__unpack(NULL,
+                         resp->body.len, resp->body.data);
```

Comment:
(style)  code indent should use tabs where possible

Reply:
your mom should use tabs where possible